test/e2e/tests/test_repository.py

```diff
@@ -0,0 +1,97 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# 	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Integration tests for the ECR Repository API.
+"""
+
+import boto3
+import pytest
+import time
+import logging
+from typing import Dict, Tuple
+
+from acktest.resources import random_suffix_name
+from acktest.k8s import resource as k8s
+from e2e import service_marker, CRD_GROUP, CRD_VERSION, load_ecr_resource
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import TestBootstrapResources, get_bootstrap_resources
+
+RESOURCE_PLURAL = "repositories"
+
+DELETE_WAIT_AFTER_SECONDS = 10
+CREATE_WAIT_AFTER_SECONDS = 10
+
+@pytest.fixture(scope="module")
+def ecr_client():
+    return boto3.client("ecr")
+
+@service_marker
+@pytest.mark.canary
+class TestEcrRepository:
```

Comment:
Sorry for being a nit, but can we change the above to just `TestRepository`? :)

Reply:
Keep them coming @jaypipes :) fixed!